hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/RpcConnection.java

```diff
@@ -126,8 +136,13 @@ protected RpcConnection(Configuration conf, HashedWheelTimer timeoutTimer, Conne
       LOG.debug("Use " + authMethod + " authentication for service " + remoteId.serviceName
           + ", sasl=" + useSasl);
     }
+
     reloginMaxBackoff = conf.getInt("hbase.security.relogin.maxbackoff", 5000);
```

Comment:
Isn't this confusing now? Consider an operator or a tech writer looking at this code. What does "maxbackoff" mean? It's not the max time we would back off (5 seconds?) when now there is "hbase.security.force.relogin.backoff" at 600000 I assume milliseconds. These things need better naming, as an operator I have no clear idea what to do here.

Reply:
While we are at it, please make these configuration keys string constants.